core/decorator.py

```diff
@@ -21,16 +22,23 @@ def __init__(self, redirect_to):
 class HttpResponseUnsupportedMediaType(HttpResponse):
     status_code = 415
 
+#replace with cache_control in django 1.10+
+def cache_page_function(function, ttl, shared_cache_maxage=None):
```

Comment:
I'm not in love with `cache_page_function`. What do you think about a more declarative name such as `add_cache_control_headers` or `add_caching_headers` which captures the idea that all it does is take a response and add headers to it?